internal/pkg/deploy/pipeline.go

```diff
@@ -166,6 +204,16 @@ func (s *GitHubSource) Owner() (string, error) {
 	return owner, nil
 }
 
+// Owner returns the repository owner portion. For example,
+// given "aws/amazon-copilot", this function returns "aws".
+func (s *BitbucketSource) Owner() (string, error) {
```

Comment:
Is this method used?

Reply:
Ah, you're right-- owner name is included in `Repository()`. Thanks! Deleted.